Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -311,6 +311,33 @@ def condition_(self, fragment, stepList, key, hasHarvest):
     '--procModifiers': 'trackingMkFit'
 }
 
+#DeepCore seeding for JetCore iteration workflow
+class UpgradeWorkflow_seedingDeepCore(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Reco' in step: stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
+    def condition(self, fragment, stepList, key, hasHarvest):
+        result = (fragment=="QCD_Pt_1800_2400_14") and ('2021' in key or '2024' in key) and hasHarvest and self.condition_(fragment, stepList, key, hasHarvest)
+        if result:
+            # skip ALCA and Nano
+            skipList = [s for s in stepList if (("ALCA" in s) or ("Nano" in s))]
+            for skip in skipList:
+                stepList.remove(skip)
+        return result
+    def condition_(self, fragment, stepList, key, hasHarvest):
+        return True
```

Comment:
This is unnecessary for this workflow and can be removed

Reply:
ok implemented